vendor_skins/Evercast/UI/window-basic-settings.cpp

```diff
@@ -835,6 +835,41 @@ OBSBasicSettings::~OBSBasicSettings()
 	EnableThreadedMessageBoxes(false);
 }
 
+static inline std::string MapFPS(QString val)
```

Comment:
How did you find all this?

Reply:
Oh, this was just a requirement - there were existing values which I wanted to map to the new ones Brad defined.